src/components/Search/index.jsx

```diff
@@ -22,41 +26,54 @@ export default function Search({ indices, changeOpenSearch }) {
       ),
     []
   );
-  const classes = useStyles();
   const [openResults, setOpenResults] = React.useState(false);
+  const [value, setValue] = useState();
   const handleClick = () => {
     changeOpenSearch(true);
     setOpenResults(true);
+    setValue();
```

Comment:
Is this necessary?

Reply:
yes because without this function then value="" permanently, thus you cannot further type in any more searches